platform-service/src/unstract/platform_service/controller/platform.py

```diff
@@ -175,8 +165,8 @@ def page_usage() -> Any:
     )
 
     try:
-        with be_db.atomic() as transaction:
-            be_db.execute_sql(query, params)
+        with db.atomic() as transaction:
```

Comment:
Just a generic doubt. We are only doing single execute statement as far as I can see. Does using atomic even makes any impact here? @hari-kuriakose @johnyrahul @muhammad-ali-e

Reply:
@ritwik-g If we are running a single query then `autocommit` should be True to maintain integrity.

Only the following are safe:
- Queries running inside Django transactions
- Queries running outside Django transactions but with `autocommit=True` (Django does this by default)

See https://docs.djangoproject.com/en/5.1/topics/db/transactions/#why-django-uses-autocommit